BreadPlayer.Views.UWP/ViewModels/ShellViewModel.cs

```diff
@@ -148,6 +150,15 @@ void HandleExecuteCmdMessage(Message message)
             Reset();
             await Player.Stop();
         }
+
+        void SaveSongToStopAfter(Message songToStopAfter)
```

Comment:
Please follow the naming convention for `MessageCallback` i.e. change the method name to `HandleSaveSongToStopAfterMessage`. Thanks.